### PR TITLE
Add content event resource to repo

### PR DESCRIPTION
**Pull request by m1yag1**
Added the content event resource to the repository.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
+include concourse/content-event-resource/Makefile
+
 .PHONY: help
 help:
 	@echo "make help              Show this help message"
 	@echo "make services          Run the services that Concourse and the pipeline requires"
 	@echo "make initdb            Restore DB dump on first run, can be also used to restore DB"
 	@echo "make sql               Connect to the cnx-db database with a psql shell"
+	@$(MAKE) help_submake --no-print-directory
```
Comment (mmulich) [inline Makefile:12]: Does this work if there is more than one submake?

Comment (m1yag1) [reply Makefile:12]: if more are added you need to make them unique... for example if there is another Makefile you'd have to change this to `help_event_resource_submake`.

File: concourse/content-event-resource/src/in_.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    event = api.get_event(api_root, event_id)
+    msg("Event Returned: {}", event["id"])
+
+    # Write out files
+    with open(os.path.join(dest_path, "id"), "w") as file:
+        file.write(str(event_id))
+
+    with open(os.path.join(dest_path, "ident_hash"), "w") as file:
+        file.write(event["ident_hash"])
```
Comment (mmulich) [inline concourse/content-event-resource/src/in_.py:22]: FYI This is our input to the bridge resource. It'll be a parameter in `get` to something like `ident_hash_file`.

**Review by mmulich — APPROVED** [review]
I like it. 👍 As coded there aren't any problems.

My understanding of the version pointer after an `out` is not solidified. You haven't coded that here, which is fine, but `check` may need adjusted if my understanding is correct, but as coded now, I see no issue.

File: concourse/content-event-resource/src/in_.py (hunk 2 above)
Comment (m1yag1) [reply concourse/content-event-resource/src/in_.py:22]: Do you want me to rename it?

**Comment by m1yag1** [comment]
> My understanding of the version pointer after an `out` is not solidified. You haven't coded that here, which is fine, but `check` may need adjusted if my understanding is correct, but as coded now, I see no issue.

Yep, not meant to be perfect and i figure adjustments will need to be made. Getting us far enough to identify the issues in the pipeline when they do come up and refactor.

Additionally, out would probably be changing the status in the service. so doing a PATCH and providing the concourse build may be a good idea. OUT could be implemented as another PR. In this case it's supposed to provide a way to "start" the pipeline.

File: Makefile (hunk 1 above)
Comment (karenc) [reply Makefile:12]: `make build-image` doesn't work in the root directory:

```
karen@yachiyo:~/src/rap-spike-concourse$ make build-image
python setup.py sdist
python: can't open file 'setup.py': [Errno 2] No such file or directory
concourse/content-event-resource/Makefile:20: recipe for target 'build-sdist' failed
make: *** [build-sdist] Error 2
```

May be we shouldn't just include the commands.

**Comment by c0ldcalamari** [comment]
Confirmed, successful pipeline set up - 
Screen Shot 2019-08-13 at 1 16 25 PM

**Pull request merged**